### DISCUSSION TITLE
F10 map toggle

### DISCUSSION THREAD

**Discussion by zero2005** [Munkwolf/dcs-community-keybinds#19] [General]
I don't use github, so I wasn't sure where to put this, but I made a command that might be useful to some:

{down = iCommandViewAWACS, up= iCommandViewCockpit, name = _('TOGGLE F10 Theater map view'), category = _('View')},

I use this as a toggle for quickly glancing at the F10 map, I use it on my Warthog on one of the toggle switches, specificially the Engine L fuel flow switch. In the up position == map on, when off == back to F1 view.

I placed it in UiLayer\Joystick\default.lua so that it is available to every plane I fly.

**Comment by Munkwolf** [comment]
Thanks for sharing! I'll add that in.

**Reply by Munkwolf** [reply]
I had this going in my local files for a while now but just realized I had never committed the change. Sorry for the delay, this is now available in the 22.7.7 release.
